dx-111 emphasized that callback is mandatory (#320)
SwedbankPay/developer.swedbankpay.com: _includes/callback-reference.md

transaction, Swedbank Pay will perform a callback to inform the payee (merchant)
about this update.

> Setting a `callbackUrl` in the HTTP `POST` API is **mandatory**. Below there
> are three example scenarios to why this is important:

Reviewer: ```suggestion
Providing a `callbackUrl` in `POST` requests is **mandatory**. Below we provide
three example scenarios of why this is important:
```

> 1. If the payer closes the payment window, the merchant will never know what

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

> 1. If the payer closes the payment window, the merchant will never know what

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

    happened to the payment if `callbackUrl` is not implemented.

> 2. If the payer stops up in a payment app such as Vipps or Swish, the payer

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

> 2. If the payer stops up in a payment app such as Vipps or Swish, the payer

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

    will never come back to the merchant. This means that the merchant won't know what

>     happened to the payment unless `callbackUrl` is implemented.

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

>     happened to the payment unless `callbackUrl` is implemented.

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

> 3. If a payer closes the browser window, a network error or

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

> 3. If a payer experiences a network error or something else happens that

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

> 3. If a payer closes the browser window, a network error or
>     something else can happen that prevents the payer from being redirected from
>     Swedbank Pay back to the merchant website. The `callbackUrl` is what ensures that
>     you receive the information about what happened with the payment.

Reviewer: ```suggestion
3. If a payer experiences a network error or something else happens that
   prevents the payer from being redirected from Swedbank Pay back to the
   merchant website, the `callbackUrl` is what ensures that you receive the
   information about what happened with the payment.
```
